client/src/store/post/reducers.ts

```diff
@@ -41,7 +41,7 @@ const defaultState = {
 
 export const postReducer = (
     state: any = defaultState,
-    action: { type: string, payload: any, loading: boolean }): any => {
+    action: { type: string, payload: any, loading: boolean, postId: string }): any => {
```

Comment:
reducer is a simple function which has 2 arguments - type: string, payload

remove loading: boolean, postId: string

Reply:
Okay. Fixed